scoper.inc.php

```diff
@@ -33,13 +33,6 @@
 
 declare(strict_types=1);
 
-use Isolated\Symfony\Component\Finder\Finder;
```

Comment:
Was this necessary? This means the Symfony polyfills are no longer excluded.

I think the canonical way to do this slightly changed though: https://github.com/humbug/php-scoper/blob/master/docs/further-reading.md#polyfills

Reply:
Given we have green builds, do we really need to exclude symfony polyfills?